tests/shared/importers/test_autoconfig.py

```diff
@@ -1,223 +0,0 @@
-import shutil
-from pathlib import Path
-from typing import Text, Set
-from unittest.mock import Mock
-
-import pytest
-from _pytest.capture import CaptureFixture
-from _pytest.monkeypatch import MonkeyPatch
-
-from rasa.shared.constants import CONFIG_AUTOCONFIGURABLE_KEYS
-import rasa.shared.utils.io
-from rasa.shared.importers import autoconfig
-
-CONFIG_FOLDER = Path("data/test_config")
-
-SOME_CONFIG = CONFIG_FOLDER / "stack_config.yml"
-DEFAULT_CONFIG = Path("rasa/shared/importers/default_config.yml")
-
-
-@pytest.mark.parametrize(
-    "config_path, autoconfig_keys",
-    [
-        (Path("rasa/cli/initial_project/config.yml"), {"pipeline", "policies"}),
-        (CONFIG_FOLDER / "config_policies_empty.yml", {"policies"}),
-        (CONFIG_FOLDER / "config_pipeline_empty.yml", {"pipeline"}),
-        (CONFIG_FOLDER / "config_policies_missing.yml", {"policies"}),
-        (CONFIG_FOLDER / "config_pipeline_missing.yml", {"pipeline"}),
-        (SOME_CONFIG, set()),
-    ],
-)
-def test_get_configuration(
-    config_path: Path, autoconfig_keys: Set[Text], monkeypatch: MonkeyPatch
-):
-    def _auto_configure(_, keys_to_configure: Set[Text]) -> Set[Text]:
-        return keys_to_configure
-
-    monkeypatch.setattr(autoconfig, "_dump_config", Mock())
-    monkeypatch.setattr(autoconfig, "_auto_configure", _auto_configure)
-
-    config = autoconfig.get_configuration(str(config_path))
-
-    if autoconfig_keys:
-        expected_config = _auto_configure(config, autoconfig_keys)
-    else:
-        expected_config = config
-
-    assert sorted(config) == sorted(expected_config)
-
-
-@pytest.mark.parametrize("config_file", ("non_existent_config.yml", None))
-def test_get_configuration_missing_file(tmp_path: Path, config_file: Text):
-    if config_file:
-        config_file = tmp_path / "non_existent_config.yml"
-
-    config = autoconfig.get_configuration(str(config_file))
-
-    assert config == {}
```

Comment:
This test has been removed because importers will return empty dict without going through auto-config now on.

Reply:
All other tests in this file have been moved to `tests/engine/recipes/test_default_recipe.py` because auto-config is recipe business.